adapters/sovrn/sovrntest/video/multi-banner-video.json

```diff
@@ -0,0 +1,184 @@
+              "h": 600
+            }
+          ]
+        },
+        "ext": {
+          "bidder": {
+            "tagid": "123456"
+          }
+        }
+      },
+    {
+      "id": "test-imp-id2",
+      "video": {
+        "mimes": [
+          "video/mp4",
+          "video/3gpp",
+          "video/x-ms-wmv"
+        ],
+        "minduration": 10,
+        "maxduration": 60,
+        "protocols": [
+          4,
+          5,
+          6,
+          8
+        ]
+      },
+      "ext": {
+        "bidder": {
+          "tagid": "654321"
+        }
+      },
+      "tagid": "654321" 
+    }
+    ],
+    "device": {
+      "ua": "test-user-agent",
+      "ip": "123.123.123.123",
+      "language": "en",
+      "dnt": 0
+    },
+    "site": {
+      "domain": "www.publisher.com",
+      "page": "http://www.publisher.com/awesome/site"
+    },
+    "user": {
+      "buyeruid": "test_reader_id"
+    }
+  },
+
+  "httpCalls": [
+    {
+      "expectedRequest": {
+        "headers": {
+          "Content-Type": ["application/json"],
+          "User-Agent": ["test-user-agent"],
+          "X-Forwarded-For": ["123.123.123.123"],
+          "Accept-Language": ["en"],
+          "Dnt": ["0"],
+          "Cookie": ["ljt_reader=test_reader_id"]
+        },
+        "uri": "http://sovrn.com/test/endpoint",
+        "body": {
+          "id": "test-request-id",
+          "imp": [
+            {
+              "id": "test-imp-id1",
+              "banner": {
+                "format": [
+                  {
+                    "w": 300,
+                    "h": 250
+                  },
+                  {
+                    "w": 300,
+                    "h": 600
+                  }
+                ]
+              },
+              "tagid": "123456",
+              "ext": {
+                "bidder": {
+                  "tagid": "123456"
+                }
+              }
+            },
+            {
+              "id": "test-imp-id2",
+              "video": {
+                "mimes": [
+                  "video/mp4",
+                  "video/3gpp",
+                  "video/x-ms-wmv"
+                ],
+                "minduration": 10,
+                "maxduration": 60,
+                "protocols": [
+                  4,
+                  5,
+                  6,
+                  8
+                ]
+              },
+              "tagid": "654321",
+              "ext": {
+                "bidder": {
+                  "tagid": "654321"
+                }
+              }
+            }
+          ],
+          "site": {
+            "domain": "www.publisher.com",
+            "page": "http://www.publisher.com/awesome/site"
+          },
+          "user": {
+            "buyeruid": "test_reader_id"
+          },
+          "device": {
+            "ua": "test-user-agent",
+            "ip": "123.123.123.123",
+            "dnt": 0,
+            "language": "en"
+          }
+        }
+      },
+      "mockResponse": {
```

Comment:
Please consider adding a response for video as well in this test case. This will allow this test case to verify that responses with multiple bids for various imps are processed correctly and will also verify the getBidType logic to make sure it matches bids to imps properly based on the imp ID

Reply:
Good point. Done.

Reply:
Good point. Done.